src/Concerns/DealsWithTenantSymlinks.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Stancl\Tenancy\Concerns;
+
+use Illuminate\Support\Collection;
+use Stancl\Tenancy\Database\Models\Tenant;
+
+trait DealsWithTenantSymlinks
+{
+    /**
+     * Get all possible tenant symlinks, existing or not (array of ['public path' => 'storage path']).
+     * This is used for creating all possible tenant symlinks and removing all existing tenant symlinks.
```

Comment:
I think this should explain a bit better why there are "possible symlinks" i.e. why there are multiple possible ones. I think it relates to the config being an array that can have multiple values, and therefore multiple symlinks?

Still, there should be one more sentence here explaining that.

Reply:
If there are two disks in the url_override config array, the tenant has two possible symlinks. So the first line of the comment should say something like:

"Get tenant symlinks (existing or not) for all disks registered in the `tenancy.filesystem.url_override` config"

Does that make it a bit more clear?

Reply:
I would *add another paragraph*, not replace the "Get all possible tenant symlinks" since that's correct and descriptive.

It should simply be clear why there can be multiple potential symlinks.

Reply:
Looks great now